parse/getast.go

```diff
@@ -23,10 +24,11 @@ type FileSet struct {
 
 // File parses a file at the relative path
 // provided and produces a new *FileSet.
-// (No exported structs is considered an error.)
+// (No structs is considered an error.)
```

Comment:
Doesn't this maintain the previous behavior if the flag to process local imports isn't provided? The documentation should make this clear, currently it makes me think that this would change the previous behaviour.